Session-3/CurrencyExchangeApp/CurrencyExchangeApp/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.CSharp.RuntimeBinder;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CurrencyConverter;
+using RestSharp;
```

Comment:
Place CurrencyExchangeApp and CurrencyConverter both projects inside same solution